### ISSUE TITLE
Remove http support of Reverseproxy Build-In Scripts

### ISSUE THREAD

**Issue by robertfausk** [projektmotor/docker-apache-load-balancer#2]
* it's currently not implemented properly but an input option (build in script is with ssl working only)
* https is web standard
* less code to maintain for us and a simplier script

@soeren-helbig WDYT?

**Comment by soeren-helbig** [comment]
@robertfausk i would prefer the "implementation-way". there are use cases where ssl isn't that funny... specially in dev env. is this some kind of a blocker for dockini? i will have a look at this... this evening ... maybe... :rabbit:

**Comment by robertfausk** [comment]
No, actually it's no blocker. Using http is offered on running the script via "Enable SSL [y|n]" but it only works with SSL enabled right now.
No need to act this evening IMO.
